test/k8sT/Policies.go

```diff
@@ -661,6 +661,7 @@ var _ = Describe("K8sPolicyTest", func() {
 			Expect(err).To(BeNil(), "Cannot get web pods")
 
 			serviceIP, port, err := kubectl.GetServiceHostPort(helpers.DefaultNamespace, "redis-master")
+			Expect(err).To(BeNil())
```

Comment:
Please add a string comment to describe the problem here, similar to how line 661 has.